WcaOnRails/app/views/registrations/register.html.erb

```diff
@@ -1,107 +1,111 @@
 <% provide(:title, I18n.t('registrations.new_registration.title', comp: @competition.name)) %>
 
 <%= render layout: "nav" do %>
-  <% opens_in_days = distance_of_time_in_words_to_now(@competition.registration_open) %>
-  <% closes_in_days = distance_of_time_in_words_to_now(@competition.registration_close) %>
-  <% unless @competition.registration_opened? %>
-    <% if @competition.registration_not_yet_opened? %>
-      <%= alert :info, t('registrations.will_open_html', days: opens_in_days, time: wca_local_time(@competition.registration_open)) %>
-    <% end %>
-    <% if @competition.registration_past? %>
-      <%= alert :danger, t('registrations.closed_html', days: closes_in_days, time: wca_local_time(@competition.registration_close)) %>
-    <% end %>
+  <% if current_user.team_member?('banned') %>
```

Comment:
Can we pull this logic out into the `user` class? I'd rather see nice friendly methods like `banned?` and `banned_until` (`banned_until` might be sufficient... if `banned_until` is `nil`, then you're not banned).

Also, this diff is terrifyingly large, it's really hard to tell what happened here. Is there any reason we can't make use of our existing `cannot_register_for_competition_reasons` infrastructure instead, and leave this `erb` file alone?